ibazel/main.go

```diff
@@ -39,6 +39,7 @@ var overrideableBazelFlags []string = []string{
 	"--config=",
 	"--copt=",
 	"--curses=no",
+        "--cxxopt",
```

Comment:
Sorry to pester, but can you switch this to being a tab instead of spaces, or run `gofmt` on it?